tests/functional/tox.ini

```diff
@@ -10,6 +10,11 @@ deps =
 
 passenv=*
 
+setenv=
+  anchore_0.6.0: PYTEST_CONTAINER = anchore/inline-scan:v0.6.0
```

Comment:
One should be `anchore/inline-scan:latest`. The CLI is expected to be used of similar release to the engine version, the versions are sync'ed so a 0.5.x CLI won't necessarily work with engine 0.6.x and vice-versa. As such, I think `latest` is sufficient here.

Reply:
I've added that and included it in the circleci configuration as well

Reply:
As I think about this, I think there is a fundamental issue here: How would one add a new CLI command and test against an unreleased anchore engine feature? As I see it here, it would break and thus not be mergeable as this config stands. I think we need to differentiate regression tests from feature tests here or build against the master branch of anchore engine or allow a tag to be specified via env vars. Am I missing something?

Reply:
That is an excellent point. These functional tests allows one to test against a running anchore-engine container, regardless of version. The difference is that `tox` is being used in the CI. 

If a developer wants to test against an unreleased anchore engine, all they need to do is run that container and expose the API port. 

There are 3 distinct workflows that can happen with this work, and I am not sure if you are referring to all of them or one of:

* CI with predefined tox environments, for example `tox -e py36-anchore_v6.0.0`
* Developer running `tox` locally with `tox -e py36-anchore_latest` 
* Developer running `pytest` directly with either an existing anchore-engine container or letting the fixture run the container for testing.